datumaro/plugins/open_images_format.py

```diff
@@ -207,10 +217,144 @@ class OpenImagesImporter(Importer):
     @classmethod
     def find_sources(cls, path):
         for pattern in [
-            OpenImagesPath.FULL_IMAGE_DESCRIPTION_NAME,
-            *OpenImagesPath.SUBSET_IMAGE_DESCRIPTION_PATTERNS,
+            OpenImagesPath.FULL_IMAGE_DESCRIPTION_FILE_NAME,
+            *OpenImagesPath.SUBSET_IMAGE_DESCRIPTION_FILE_PATTERNS,
         ]:
             if glob.glob(osp.join(glob.escape(path), OpenImagesPath.ANNOTATIONS_DIR, pattern)):
                 return [{'url': path, 'format': 'open_images'}]
 
         return []
+
+class OpenImagesConverter(Converter):
+    DEFAULT_IMAGE_EXT = '.jpg'
+
+    @contextlib.contextmanager
+    def _open_csv_annotation(self, file_name, field_names):
+        absolute_path = osp.join(self._save_dir, OpenImagesPath.ANNOTATIONS_DIR, file_name)
+
+        with open(absolute_path, 'w', encoding='utf-8', newline='') as f:
+            yield csv.DictWriter(f, field_names)
+
+    def apply(self):
+        annotations_dir = osp.join(self._save_dir, OpenImagesPath.ANNOTATIONS_DIR)
+
+        os.makedirs(annotations_dir, exist_ok=True)
+
+        self._save_categories()
+        self._save_label_category_parents()
+        self._save_subsets()
+
+    def _save_categories(self):
+        with self._open_csv_annotation(
+            OpenImagesPath.V5_CLASS_DESCRIPTION_FILE_NAME, ['LabelName', 'DisplayName'],
+        ) as class_description_writer:
+            # no .writeheader() here, since we're saving it in the V5 format
```

Comment:
As an enhancement, you can add an optional argument for the format, which will determine the version of the exported dataset. What do you think about it?

Reply:
I _could_, but... is there any reason to do so? It could just be unnecessary flexibility.

(Incidentally, the main reason I used the V5 format for this file is so that I wouldn't need to include the dataset name in the file name. It would be weird if the user could import OIDv5, then export it and end up with `oidv6-class-descriptions.csv`, even though the data is still from v5.)

Reply:
Finally, can we import and export in one format?

Reply:
This _is_ one format; just different versions. We can't use the same version as was used during import, since that information is not preserved anywhere (and the input could be a mishmash of different versions anyway).